src/metabase/embed/settings.clj

```diff
@@ -6,18 +6,26 @@
    [metabase.analytics.snowplow :as snowplow]
    [metabase.api.common :as api]
    [metabase.models.setting :as setting :refer [defsetting]]
-   [metabase.public-settings :as public-settings]
    [metabase.util.embed :as embed]
    [metabase.util.i18n :as i18n :refer [deferred-tru]]
    [toucan2.core :as t2]))
 
 (defsetting embedding-app-origin
-  (deferred-tru "Allow this origin to embed the full {0} application"
-                (public-settings/application-name-for-setting-descriptions))
+  (deferred-tru "Allow this origin to embed the full Metabase application")
   :feature    :embedding
+  :export?    false
```

Comment:
I guess `:export? false` was assumed, I added here to match the new value in `embedding-app-origins-sdk` and to make them consistent.

This is needed otherwise the serialization test in `enterprise/backend/test/metabase_enterprise/serialization/v2/round_trip_test.clj` will fail.